src/pudl/etl/check_foreign_keys.py

```diff
@@ -0,0 +1,64 @@
+"""A command line interface (CLI) to check foreign key constraints in the PUDL database.
+
+Assets are loaded into the ``pudl.sqlite`` in parallel so foreign key constraints can
```

Comment:
Are they really loaded in parallel? I thought SQLite could only deal with a single write operation at a time and it was jjust that they're being written potentially out of order / with no regard for dependencies between them.

Reply:
You're correct they aren't loaded in parallel. The dataframes are computed in parallel but they are written to the database one at a time. The [database is locked](https://docs.sqlalchemy.org/en/14/dialects/sqlite.html#database-locking-behavior-concurrency) when a write operation is happening. I updated the doc string to explain how this works.